src/cloudevent/partials/alerts/alerts-on-alert-published.ts

```diff
@@ -0,0 +1,32 @@
+import {DeepPartial, MockCloudEventPartials} from '../../types';
+import {FirebaseAlertData} from 'firebase-functions/lib/v2/providers/alerts';
+import {CloudEvent, CloudFunction} from 'firebase-functions/lib/v2';
+import {getEventType, PROJECT_ID} from '../helpers';
+
+export const alertsOnAlertPublished:
+  MockCloudEventPartials<FirebaseAlertData, FirebaseAlertData<any>> = {
+  generatePartial(cloudFunction: CloudFunction<FirebaseAlertData>): DeepPartial<CloudEvent<FirebaseAlertData<any>>> {
+    const source = `//firebasealerts.googleapis.com/projects/${PROJECT_ID}`;
+
+    return {
+      source,
+      type: getEventType(cloudFunction),
+      data: getOnAlertPublishedData()
+    };
+  },
+  match(cloudFunction: CloudFunction<FirebaseAlertData>): boolean {
+    return getEventType(cloudFunction) === 'google.firebase.firebasealerts.alerts.v1.published';
+  },
+};
+
+/** Alert Published Data */
+
+function getOnAlertPublishedData(): FirebaseAlertData<any> {
+  const now = new Date().toISOString();
+  return ({
+    // '@type': 'type.googleapis.com/google.events.firebase.firebasealerts.v1.AlertData',
```

Comment:
Why commented out?

Reply:
+1. Why is this commented out?

Reply:
Because live responses include it... but the typing does a strange `T extends ['@type']` thing. Wasn't sure if the best solution was to simply copy the typing.. :\